pdm.lock

```diff
@@ -42,6 +42,16 @@ files = [
     {file = "asttokens-2.4.1.tar.gz", hash = "sha256:b03869718ba9a6eb027e134bfdf69f38a236d681c83c160d510768af11254ba0"},
 ]
 
+[[package]]
+name = "attrs"
+version = "23.2.0"
+requires_python = ">=3.7"
+summary = "Classes Without Boilerplate"
+files = [
+    {file = "attrs-23.2.0-py3-none-any.whl", hash = "sha256:99b87a485a5820b23b879f04c2305b44b951b502fd64be915879d77a7e8fc6f1"},
+    {file = "attrs-23.2.0.tar.gz", hash = "sha256:935dc3b529c262f6cf76e50877d35a4bd3c1de194fd41f47a2b7ae8f19971f30"},
+]
+
```

Comment:
Hmm, I'm confused why we have these additions to the `pdm.lock` file. Perhaps I missed this earlier on in the PR. Looks like you haven't added any imports for `attrs`, `referencing`, or `rpds-py`, so these changes should be discarded, I think.

Reply:
Sorry I confused `referencing` (which I'm using at work) with `packaging`